index.js

```diff
@@ -14,7 +14,13 @@ const coolOff = mon => {
 	}
 }
 
-const getFlavourText = pokedexEntry => pokedexEntry.flavor_text_entries.find(entry => entry.language.name === 'en' && entry.version.name === 'alpha-sapphire').flavor_text;
+const getFlavourText = pokedexEntry => {
+	if (getName(pokedexEntry) === 'Pidgey'||'Rattata') {
```

Comment:
Needs to be `if (getName(pokedexEntry) === 'Pidgey'|| getName(pokedexEntry) === 'Rattata')` or `if (getName(pokedexEntry).match(/Pidgey|Rattata/))`